lib/gitlab/ci/templates/pipeline/GradleInstall4JPipeline.yml

```diff
@@ -28,10 +28,11 @@ before_script:
   script:
     # Only publish when building develop or a release branch
     - if [[ '$CI_COMMIT_BRANCH =~ $DEV_OR_RELEASE_REGEX' ]]; then PUBLISH="publish"; else PUBLISH="-x publish"; fi
-    - echo "Build with SNAPSHOT='$SNAPSHOT' JDK_SELECTOR='$JDK_SELECTOR' PUBLISH='$PUBLISH' using flags '$GRADLE_FLAGS'"
-    - ./gradlew $GRADLE_FLAGS $SNAPSHOT $JDK_SELECTOR -Pi4jLicenseKey=${I4J_LICENSE_KEY} $INSTALLER_GRADLE_COMMANDS $PUBLISH
+    - echo "Build with SNAPSHOT='$SNAPSHOT_FLAGS' JDK_SELECTOR='$JDK_SELECTOR' PUBLISH='$PUBLISH' using flags '$GRADLE_FLAGS'"
```

Comment:
If we change the variable names we shouldn't merge this back into release/1.2. It's too hard to say if there are any other projects referencing the old keywords

Reply:
I can do that or switch it back, but the variable is created in this stage, not the normal variable section, so I figured it would be ok

Reply:
Yea your variable name change is better, but we would need to roll to a 1.3 branch, and notify teams to update from 1.2 to 1.3 if they want. Up to you @AndrewTrow if you want to roll to 1.3 or switch the variable back.